include/ocpp/v201/database_handler.hpp

```diff
@@ -172,16 +172,37 @@ class DatabaseHandler : public common::DatabaseHandlerCommon {
     /// charging profiles
 
     /// \brief Inserts or updates the given \p profile to CHARGING_PROFILES table
-    void insert_or_update_charging_profile(const int evse_id, const v201::ChargingProfile& profile);
+    void insert_or_update_charging_profile(
+        const int evse_id, const v201::ChargingProfile& profile,
+        const ChargingLimitSourceEnum charging_limit_source = ChargingLimitSourceEnum::CSO);
```

Comment:
@marcemmers @Pietfried where did these enums come from? I see that they were added in https://github.com/EVerest/libocpp/commit/398f0c5e3465d49a27524d708a8e8281461e69fa but I don't see these particular strings defined in the OCPP spec.

If they are, can you point me to where they are defined?
If not, can you clarify where the enums are intended to be used? For example, I don't understand the difference between "CSO" and "SO", or what "Other" will be used for

Reply:
`ChargingLimitSourceEnumType` is defined in the OCPP-201_part2_specification_edition2.pdf file, on page 409 (3.11 ChargingLimitSourceEnumType)